test/jdk/tools/jpackage/apps/ChildProcessAppLauncher.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.io.File;
+import java.io.IOException;
+
+public class ChildProcessAppLauncher {
+
+    private static final String FS = File.separator;
+
+    public static void main(String[] args) throws IOException {
+        String systemPath = System.getenv("SystemRoot");
+        if (systemPath != null) {
+            String calcPath = systemPath + FS + "system32" + FS + "calc.exe";
+            ProcessBuilder processBuilder = new ProcessBuilder(calcPath);
+            Process process = processBuilder.start();
+            System.out.println("Calc id=" + process.pid());
+            System.exit(0);
+        }
```

Comment:
The better alternative would be:
```String calcPath = Path.of(System.getenv("SystemRoot"), "system32", "calc.exe").toString();```

This way NPE will be thrown if `SystemRoot` env variable is not set instead of silent exit, and `FS` field is not needed.